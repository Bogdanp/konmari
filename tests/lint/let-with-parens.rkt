#lang racket/base

(let ((x 1)) x)
