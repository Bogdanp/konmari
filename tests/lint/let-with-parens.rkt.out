/let-with-parens.rkt:3:7:warning:bindings within a let should be surrounded by square brackets
